modules/replicator/src/main/java/org/apache/ignite/internal/replicator/configuration/ReplicationConfigurationSchema.java

```diff
@@ -40,4 +40,14 @@ public class ReplicationConfigurationSchema {
     @Value(hasDefault = true)
     @Range(min = 1000)
     public long rpcTimeout = TimeUnit.SECONDS.toMillis(60);
+
+    /** The interval in milliseconds that is used in the beginning of lease granting process. */
+    @Value(hasDefault = true)
+    @Range(min = 0)
+    public long agreementAcceptanceInterval = 120_000;
+
+    /** Lease holding interval. */
+    @Value(hasDefault = true)
+    @Range(min = 0)
```

Comment:
It shouldn't be zero, at lease it should be greater (preferable several times) than LeaseUpdater#UPDATE_LEASE_MS (500 ms). I would suggest 2000, but it's discussible. Also we should set the max value, maybe default of leaseAgreementAcceptanceTimeLimit?